### DISCUSSION TITLE
Help wit a very basic Actions / workflow

### DISCUSSION THREAD

**Discussion by AntonOks** [community/community#18931] [Actions]
Why is this [very basic workflow throwing an error](https://github.com/AntonOks/cronactions/runs/6956928257)? If I'm not totally wrong, this "just worked" in some of ma workflows on different places before...

**Comment by melvincwng** [comment]
Hi @AntonOks, perhaps you can find answers to your issue in this GitHub Support page about 'Actions' - https://support.github.com/features/actions

Otherwise, perhaps you could raise a question in the Github Community Forums or raise a support ticket.

Hope this helps

**Reply by AntonOks** [reply]
thanks for your hint. I've found my mistake by now

**Comment by thuongtruong1009** [comment]
Hi @AntonOks 
* I think that you may miss the time cron or branch for action
* It is like that: 
![image](https://user-images.githubusercontent.com/71834167/174524108-6020cbc7-fd5a-4850-b0b6-bc34160fde82.png)
* Or you can express with the community for answer https://support.github.com/features/actions
* Hope useful for you!

**Reply by AntonOks** [reply]
thanks for your hint. I've found my mistake by now

**Reply by thuongtruong1009** [reply]
> thanks for your hint. I've found my mistake by now

If you think my comment useful, please mark them as answered for everyone to learn with you. Thank you very much! 🥰
